figures/README.md

```diff
@@ -0,0 +1,32 @@
+# Create color palettes and plot theme
+
+### Usage
+
+The `create_color_palette_project.R` script is intended to be run via the command line from the top directory of the repository as follows:
+
+```
+Rscript create_color_palette_project.R
+```
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `create_color_palette_project.R`: define `root_dir`
```

Comment:
Again, for extra clarity, maybe expand to say: define `root_dir` for your specific project path

Reply:
The `root_dir` is the directory where the user has cloned the repo.
Could the following example is clear?

Example:
```
/Users/my_username/Documents/sc-rna-seq-snap
```

Reply:
For this one, I was actually meaning just putting the directions as "define `root_dir` for your specific project path". But if you want to provide that example, that works I think

Reply:
This has now been addressed here https://github.com/stjudeDNBBinfCore/sc-rna-seq-snap/commit/5d50679872637b1bafe945e5852306923e164ccf